bluepysnap/circuit.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, config):
         Returns:
             Circuit: A Circuit object.
         """
-        self._circuit_config_path = config
+        self._circuit_config_path = str(config)
```

Comment:
It seems safer to add `.absolute()` as it's done in Simulation

Reply:
Agreed -> done